### ISSUE TITLE
build error with benchmark-node-micro-benchmarks CI job

### ISSUE THREAD

**Issue by mscdex** [nodejs/build#1690]
"make[1]: execvp: printf: Argument list too long" when building SSL?

[Here](https://ci.nodejs.org/view/Node.js%20benchmark/job/benchmark-node-micro-benchmarks/289/console) is the benchmark job console output.

**Comment by mhdawson** [comment]
A few questions:

1) did you select a specific set of tests to run?
2) do you know if this is a recent issue?

@uttampawar is this something you might be able to help look at?

**Comment by refack** [comment]
FTR, this is a part of the error:
```
In file included from ../deps/icu-small/source/i18n/collationdata.h:24:0,
                 from ../deps/icu-small/source/i18n/collationsets.cpp:23:
../deps/icu-small/source/common/normalizer2impl.h: In member function ‘uint16_t icu_63::Normalizer2Impl::getNorm16(UChar32) const’:
../deps/icu-small/source/common/normalizer2impl.h:267:43: warning: enumeral and non-enumeral type in conditional expression [-Wextra]
         return U_IS_LEAD(c) ? INERT : UCPTRIE_FAST_GET(normTrie, UCPTRIE_16, c);
                                           ^
make[1]: execvp: printf: Argument list too long
make[1]: *** [/home/iojs/build/workspace/benchmark-node-micro-benchmarks/benchmarking/experimental/benchmarks/community-benchmark/node/out/Release/obj.target/deps/openssl/libopenssl.a] Error 127
make[1]: *** Waiting for unfinished jobs....
make: *** [node] Error 2
```

I was looking into this, and it seems that because the build directory is long
```
/home/iojs/build/workspace/benchmark-node-micro-benchmarks/benchmarking/experimental/benchmarks/community-benchmark/node/
```
when `make` needs to create the command line to link all the obj files, and it uses absolute paths, the result is vvveeerrrryyyy long.

**Comment by refack** [comment]
https://ci.nodejs.org/job/benchmark-node-micro-benchmarks/
tl;dr it now builds both binaries. Actual benchmarks "should" work.
